packages/mobile/src/consumerIncentives/ConsumerIncentivesHomeScreen.tsx

```diff
@@ -101,89 +49,65 @@ export default function ConsumerIncentivesHomeScreen(props: Props) {
   const onLearnMore = () => navigate(Screens.WebViewScreen, { uri: CELO_REWARDS_LINK })
```

Comment:
Here we need to change the URI for the new link for terms and conditions. We need to add the link to the [branding repo](https://github.com/clabs-co/valora-app-branding) as well. The new URI is `https://celo.org/save-terms-and-conditions`

Reply:
Ah I see, sorry I got confused about the different links and which one is opened where